test/language/expressions/async-generators/expression-yield-promise.js

```diff
@@ -0,0 +1,48 @@
+// Copyright 2017 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+author: Caitlin Potter <caitp@igalia.com>
+esid: pending
+description: >
```

Comment:
I'm open to suggestions on better description text, but otherwise I'm leaving these as-is, and the web runner can just deal with it